examples/python-library-gcp-keyring/pyproject.toml

```diff
@@ -0,0 +1,79 @@
+# Introduction:
+#
+# This is a small example that builds and uploads a `.whl` to a custom pypi registry
+# If you want to upload to PyPI just replace the `--repository-url`
+# To use with GCP, make sure you've run a `gcloud auth`
+
+# Using the custom registry:
+#
+# To use the custom GCP registry, unfortunately you have to have keyring installed globally
+# Because `uv` which we use for PyPI resolution uses a subprocess to call `keyring`
+# Also you need to add `oauth2accesstoken@` to the url e.g.
+# `https://oauth2token@europe-west4-python.pkg.dev/test-artifact-registry-423208/test/`
+#
+# Example of a feature that uses this:
+#
+# [feature.hi.pypi-options]
+# index-url = "https://oauth2accesstoken@europe-west4-python.pkg.dev/test-artifact-registry-423208/test/"
+#
+# [feature.hi.pypi-dependencies]
+# hi = { version = "==0.1.0" }
+#
+# To Install:
+# Use `pixi install --pypi-keyring-provider subprocess` or configure in global configuration
+
+[project]
+name = "hi"
+version = "0.1.0"
+description = "Add a short description here"
```

Comment:
```suggestion
name = "python-library-gcp-keyring"
```

Reply:
That will also make the wheel be called that, I thought it was kinda nice to have the environment seperate from the library. WDYT?

Reply:
Changed it!